pages/docs/widgets/variants/button.en.mdx

```diff
@@ -10,29 +10,39 @@ import { RequestToken } from '@/components/intermediary/request-token'
 
 This widget renders only a button, which when clicked, opens our calculator.
 
-It doesn’t required any equipment data to work, so you can place this button anywhere on your website.    
+It doesn’t required any equipment data to work, so you can place this button anywhere on your website.  
 The widget has several configuration parameters, which you can read about below.
 
 ## Getting started
 
 Add a piece of HTML to your web page with class `beequip-button`, in a location where you want the button to render.
 
 ```html
-<a href="https://lease.beequip.nl" class="beequip-button">Lease equipment bij Beequip</a>
+<a href="https://lease.beequip.nl" class="beequip-button"
+    >Lease equipment bij Beequip</a
+>
 ```
 
 Next, add the [required parameters](#required-parameters) with data-attributes, as described below.
 
 ```html
-<a href="https://lease.beequip.nl" class="beequip-button" data-token="Xhpx7QT6xRF9ZHyABrxmm12Q">
+<a
+    href="https://lease.beequip.nl"
+    class="beequip-button"
+    data-token="Xhpx7QT6xRF9ZHyABrxmm12Q"
+>
```

Comment:
In case of code formatting, the rule "Always leave code better than you found it" is very subjective. Your change doesn't make the code necessarily better than what is was. Even worse: now you've introduced a difference in code examples between this page and the documentation of the calculator widget. So my suggestion would be to focus your time and energy on the task at hand, and only change formatting if it's truly wrong.

Reply:
Manually reverted the changes